content/pages/constitution.md

```diff
@@ -385,7 +385,65 @@ Committee office, M3 1003.
 5. Once the committee has been chosen, the Chair of the WiCS Committee must
    ratify the new committeee.
 
-## iii. Standing Committees ##
+## iii. Resignations and Removal of Members ###
+
+1.  Members may resign from the committee at any time by providing notice to all
+    current committee members.
+
+2.  Any member of the committee may initiate the process for removing a
+    committee member, provided the following:
+
+    1. Committee members may only be removed from the committee if they commit
+       serious violations of policies. These include failure to uphold
+       responsibilities assigned by the Constitution and committee, unauthorized
+       actions on behalf of the committee that lead to negative outcomes, and
+       serious violations of policy and procedure that run contrary to committee
+       values;
```

Comment:
This kind of says: "Serious violations of policy including serious violations of policy" it sounds redundant.

Could the last clause be "violations of procedure or other actions that run contrary to committee values" ?